index.html

```diff
@@ -1,5 +1,15 @@
 <script src="https://unpkg.com/vue/dist/vue.js"></script>
 
-<div id="app"></div>
+<div id="app">
+  <p>{{ text }}</p>
+
+  <p>
+    <a v-bind:href="linkTo" target="_blank">DuckDuckGo</a>
+  </p>
+
+  <p>
+    <a :href="linkTo" target="_blank">DuckDuckGo</a>
```

Comment:
Note this is equivalent to line 7, but included here to illustrate this shorthand. `v-bind:href` is equivalent to `:href`. This syntax is also very readable so I'm going to use this shorthand from here on.